### ISSUE TITLE
Crash in MATLAB launcher on develop branch

### ISSUE THREAD

**Issue by mael25** [cyberbotics/webots#3132]
**Describe the Bug**
The launcher crash when trying to to get the environment of the project on the `develop` branch.

**Steps to Reproduce**
1. Launch any MATLAB controller

**Expected behavior**
Manage to find the environment path.

**Screenshots**
![error_matlab](https://user-images.githubusercontent.com/38250944/120671983-2fbb3780-c492-11eb-8f58-e7238bdee634.png)

**Additional context**
The function `wbu_system_getenv` does not exist anymore in `lib/controller/matlab`.

**Comment by mael25** [comment]
It works well when replacing l.156-157 of `lib/controller/matlab/launcher.m` with:
```
  WEBOTS_PROJECT_UTF8 = calllib('libController', 'wbu_system_getenv', 'WEBOTS_PROJECT');
  WEBOTS_PROJECT = calllib('libController', 'wbu_system_short_path', WEBOTS_PROJECT_UTF8);
```

But then other functions are not defined for MATLAB:
![error_matlab_2](https://user-images.githubusercontent.com/38250944/120673917-e5d35100-c493-11eb-9cde-8315d465ba4e.png)


So there must be an issue with the adaptation of the functions for MATLAB.

**Comment by BenjaminDeleze** [comment]
Do you have the `wbu_system_getenv.m` file in `webots/lib/controller/matlab`?
Are you on Windows or Linux?

**Comment by mael25** [comment]
No, it is note ther anymore. I use Linux.

**Comment by BenjaminDeleze** [comment]
Can you try to `make` and see if it is still missing please?

**Comment by mael25** [comment]
Yes, it seems to still be missing...

**Comment by BenjaminDeleze** [comment]
ok, I cannot reproduce it.
In your make process, did you have these lines printed
` Matlab controller library (release)  
generating matlab
Unable to find python, skipping matlab API`

it should appear between:
` Java controller library (release)`

and 

` Python 2.7 controller library (release) `

**Comment by mael25** [comment]
Yes I have `Unable to find python, skipping matlab API` , so I think we can close this issue.

**Comment by BenjaminDeleze** [comment]
I opened a PR (#3137) about that.

Can you test it to see if it fixes the problem?

**Comment by mael25** [comment]
Solved by #3137
